README.md

```diff
@@ -385,32 +379,33 @@ Bash???
 
 ## Git Workflow
 
-Git Hooks?? Husky? @gmoura???
+* commitizen
+* husky
+* lint-staged
+* semantic-release
+
+_Note_ the above tools are used by [admission.laboratoria.la](https://github.com/Laboratoria/admission.laboratoria.la) and [lms.laboratoria.la](https://github.com/Laboratoria/lms.laboratoria.la)
 
 See [GIT_WORKFLOW.md](./GIT_WORKFLOW.md).
 
 ***
 
 ## Dependency management
 
-When and how to upgrade deps.
-
-Semver (`^`, `~` and pinning deps).
-
-`npm` vs `yarn`
-
-Lock files (when to add to git and when not to).
+* [Dependabot](https://dependabot.com/)
```

Comment:
most of our repos related to products used dependabot to automate dependency updates